syntax = "proto2";

package protobrain.proto;

message SparseDistributedRepresentation {
  repeated int32 shape = 1;
  repeated int32 on_bits = 2;
}
